randovania/games/dread/json_data/Artaria.txt

```diff
@@ -624,7 +626,13 @@ Extra - asset_id: collision_camera_005
   > Door to White EMMI Introduction
       Trivial
   > Tile Group (POWERBEAM)
-      Grapple Beam or Spider Magnet or Space Jump or Simple IBJ
+      Any of the following:
+          Grapple Beam or Spider Magnet or Space Jump or Simple IBJ
```

Comment:
Grapple only should be paired with beginner movement here or wall jump. it requires you to either wall jump or jump out of the tether.